regression/goto-harness/chain.sh

```diff
@@ -23,5 +23,7 @@ if [ -e "${name}-mod.gb" ] ; then
   rm -f "${name}-mod.gb"
 fi
 
-$goto_harness "${name}.gb" "${name}-mod.gb" --harness-function-name $entry_point ${args} 
+$cbmc --show-goto-functions "${name}.gb"
```

Comment:
Do you need this information for the tests or is this just intended to make debugging a bit easier?

Reply:
Not need, so I guess debugging.